bulkRoleManager/bulkRoleManager.py

```diff
@@ -205,6 +228,144 @@ async def makeDE(self, ctx, *userList):
         await ctx.send(message)
     
+    @commands.command()
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def makePermFA(self, ctx, tier: str, *userList):
+        """Makes each member that can be found from the userList a permanent Free Agent for the given tier"""
+        role_names_to_add = ["permFA", "League", tier, "{0}FA".format(tier)]
+        roles_to_add = []
+        tiers = await self.team_manager_cog.tiers(ctx)
+        for role in ctx.guild.roles:
+            if role.name in role_names_to_add:
+                roles_to_add.append(role)
+                role_names_to_add.remove(role.name)
+                if role.name == "League":
+                    leagueRole = role
+        
+        if role_names_to_add:
+            await ctx.send(":x: The following roles could not be found: {0}".format(", ".join(role_names_to_add)))
+            return False
+
+        empty = True
+        notFound = 0
+        had = 0
+        added = 0
+        message = ""
+        for user in userList:
+            try:
+                member = await commands.MemberConverter().convert(ctx, user)
+            except:
+                message += "Couldn't find: {0}\n".format(user)
+                notFound += 1
+                continue
+            if member in ctx.guild.members:
+                empty = False
+                if leagueRole in member.roles:
+                    tier_changed = False
+                    old_tier_role = await self.team_manager_cog.get_current_tier_role(ctx, member)
+                    
+                    if (old_tier_role in member.roles and (not old_tier_role in roles_to_add)) or not old_tier_role:
+                        tier_changed = True
+                    elif old_tier_role in member.roles and old_tier_role in roles_to_add:
+                        had += 1
+                        added -= 1  # remove double count of had/added
+                        
+
+                    if tier_changed:
+                        action = "assigned"
+                        if old_tier_role:
+                            old_tier_fa_role = self.team_manager_cog._find_role_by_name(ctx, "{0}FA".format(old_tier_role.name))
+                            rm_roles = [old_tier_role, old_tier_fa_role]
+                            await member.remove_roles(*rm_roles)
+                            action = "promoted"
+                        tier_change_msg = ("Congrats! Due to your recent ranks you've been {0} to our {1} tier! "
+                        "You'll only be allowed to play in that tier or any tier above it for the remainder of this "
+                        "season. If you have any questions please let an admin know."
+                        "\n\nIf you checked in already for the next match day, please use the commands `[p]co` to check "
+                        "out and then `[p]ci` check in again for your new tier.").format(action, tier)
```

Comment:
Quick thing but can you add the word "to" after `[p]ci` here?

Reply:
yes!

Reply:
Done.